### ISSUE TITLE
running "process-compose" with the same port twice makes the second instance inaccessible via REST API

### ISSUE THREAD

**Issue by adrian-gierakowski** [F1bonacc1/process-compose#68]
## Defect

When multiple instance of `process-compose` are ran concurrently, only the first instance is accessible via REST API. This can lead to confusion, if one forgets that an instance is already running in the background and starts a new one, and the tries to access it via REST or `process-compose` command (like `process-compose process list` etc). 

This is also the case if any other process is already using the port process-compose is supposed to use. 

IMHO, `process-compose` should exit with an error if it cannot bind the the port. If one doesn't intend to use the REST interface, a flag to disable the REST server could be provided. 

#### Version of `process-compose`:

```
Process Compose
Version:        v0.51.4
Commit:         e3cc52e
```

#### OS environment:

nixos

**Comment by F1bonacc1** [comment]
Good catch.
Will be fixed in the next release.

**Comment by F1bonacc1** [comment]
Should be fixed in v0.60.0
Feel free to reopen if it doesn't work as expected.
